### ISSUE TITLE
Sell Coins

### ISSUE THREAD

**Issue by awilbourn** [bchavez/Coinbase#13]
Is there any work going forward to build in to sell coins and have them deposited to a bank account associated with a coinbase account? If not, what is the process to be able to contribute to the base? I have not ever done that before, but as you have helped with other parts would not mind giving some back.

**Comment by bchavez** [comment]
@awilbourn There's some documentation on how to "send money" (aka bitcoins) here:
https://github.com/bchavez/Coinbase#send-money

In regards to how to convert bitcoins to actual government issued currency; you'd need an exchange like Coinbase or BitPay that can link to your bank account and do a conversion from BTC to USD. You can't directly send money to a US bank account unless you have access to the ACH network.

**Comment by awilbourn** [comment]
I was asking more about the Coinbase method 
https://developers.coinbase.com/api#sells, so allows you to use their 
API to initiate a sell of coins received. The project I need this for 
they want to initiate a sell as soon as they get the order to ensure 
they get the same rate they just received. There is the built in daily 
sell feature, but depending on the current rate could change I suppose, 
I am no expert on this, but the person that asked me to integrate made 
this a feature they wanted me to work on. So that is why I was asking 
how the process of contributing works for this project so I could add in 
the extra call and response. If you are able to add it and release a new 
version that would be great, but do not want to consume your time. If 
the core elements of building a request was exposed, then I could just 
do it in my app and not have to expose it specifically. I suppose if I 
used reflection I could get to the internals, but really would just 
rather add to the wrapper the call, or make the internal protected 
methods exposed to give a little more flexibility.

------ Original Message ------

**Comment by bchavez** [comment]
@awilbourn 

Please try compiling the latest commit. I've added basic support for the "sell" API call.

I've tested it against the call against `sandbox`. If you can confirm that it also works for you I can push to NuGet.

Thanks,
Brian

**Comment by awilbourn** [comment]
I just looked at the code and it looks like what I ended up doing myself 
with a copy of the code, slightly different as I really did not care if 
the property has underscores or not in the name of them, so no need to 
set the JsonProperty for the name in JSON.

I appreciate your efforts, you have done a great job! This would mean I 
could just get the latest from nuget and have what I need and not have 
to include a separate project.

------ Original Message ------
